packages/back-end/src/models/SavedGroupModel.ts

```diff
@@ -20,38 +26,27 @@ const savedGroupSchema = new mongoose.Schema({
   dateUpdated: Date,
   values: [String],
   source: String,
```

Comment:
It is deprecated.  Since I'm using this field in the JIT migration, I thought it might be safest to leave it in the mongoose schema, even if it's not being set or queried anymore.